packages/calcite-components/src/components/table/resources.ts

```diff
@@ -0,0 +1,13 @@
+export const CSS = {
+  bordered: "bordered",
+  simple: "simple",
+  borderedZebra: "bordered-zebra",
```

Comment:
I wonder if these should be reversed? `zebra-bordered` & `zebra`? Do we need the `simple`??

Reply:
Don't love `simple` naming either - but need some terminology for these four somehow. cc @mitc7862 any ideas? Solid vs outline? Plain? Booooring?

Reply:
cc @SkyeSeitz @ashetland for thoughts? is "solid" better than plain?

Reply:
Could `bordered` and `zebra` just be boolean props instead of the `appearance` prop?

Reply:
Yeah, this is a good idea... will adjust.

Reply:
Updated